src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partnerGender = () => (sex === 'f' ? 'husband' : 'wife');
```

Comment:
I think function is not necessary here
```suggestion
  const partnerGender = sex === 'f' ? 'husband' : 'wife';
```